### PR TITLE
Mepts 1002 | TX_CURR Diff Ariel Db

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

**Review by ningosi — APPROVED** [review]

**Comment by ningosi** [comment]
@helderjosue can you please pull in the 2.x_develop details in this branch, looks like the tests need to pass before the merge button is enable

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**
